spec/parser_spec.lua

```diff
@@ -1109,4 +1108,12 @@ describe("Pallene parser", function()
         assert_expression_syntax_error([[ foo as ]],
             "Expected a type for the cast expression")
     end)
+
+    it("catches break statements outside loops", function()
```

Comment:
It is allowed, just like in Lua

IIRC, the reason that Lua has the restriction with the return statements having to go at the end is to avoid an ambiguity between `return f(x)` and `return; f(x)` when the `f(x)` is on the other line.

Reply:
We could have a test case for that in coder_spec.